packs/orion/actions/ncm_config_download.py

```diff
@@ -27,6 +27,10 @@ def run(self, node, platform, configs):
         results = {}
         node_ids = []
 
+        # Split into an Array, as enum and array type does not play nice
+        # in the WebUI.
+        configs = configs.split(",")
```

Comment:
There is a limited number of config names supported on the NCM side, which is why I went with an enum so it can be extended hopefully when more are supported. However not all devices will support the two supported config names. so it's not a good idea to force it to going with an immutable parameter.